changelogs/unreleased/add-four-digit-versioning-module-tool.yml

```diff
@@ -0,0 +1,4 @@
+---
+description: Add four digit versioning for module tool (release).
```

Comment:
I think we want to add an entry to the `minor-improvement` section of the changelog.